osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -209,11 +219,11 @@ public void PlaceOnPlate(DrawableCatchHitObject fruit)
 
             if (hitLighting.Value)
             {
-                AddInternal(new HitExplosion(fruit)
-                {
-                    X = fruit.X,
-                    Scale = new Vector2(fruit.HitObject.Scale)
-                });
+                var hitExplosion = hitExplosionPool.Get();
```

Comment:
might consider explicitly typing this (i think that's what we originally decided on)
```suggestion
                HitExplosion h = hitExplosionPool.Get();
```
maybe